app/main.py

```diff
@@ -1,13 +1,17 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {"line": error["line_number"],
+            "column": error["column_number"],
+            "message": error["text"],
+            "name": error["code"],
+            "source": "flake8"}
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    return {"errors": [format_linter_error(error) for error in errors],
+            "path": file_path,
+            "status": "passed" if errors == [] else "failed"}
```

Comment:
You do not need to evaluate an empty list, it will be faulty by default. Change your code as follows:
```suggestion
            "status": "failed" if errors else "passed"}
```

Reply:
done